tox.ini

```diff
@@ -23,6 +23,7 @@ changedir = {toxinidir}/cfgov
 commands =
     coverage erase
     coverage run --source='.' manage.py test {posargs}
+    flake8
```

Comment:
Minor request: could we add the `flake8` command to `run_travis.sh` instead of as part of tox? Linting seems different to me than unit testing. The frontend code has them set up as separate gulp tasks.

In addition, when iterating on code, I find it useful to only test the specific code I'm working on, for example repeated invocations of `tox -e fast path.to.my.tests`. This is a quick way to do test-driven development. The change here runs flake8 on the _entire_ codebase even if tox is being invoked on a single file.